client/webserver/site/src/html/wallets.tmpl

```diff
@@ -23,20 +23,22 @@
 {{define "actionButtons"}}
   {{$w := .Wallet}}
   {{if $w}}
-  {{$ready := and $w.Running $w.Open}}
-  <button data-action="connect"{{if $w.Running}} class="d-hide"{{end}}>[[[Connect]]]</button>
-  <button data-action="unlock"{{if $w.Open}} class="d-hide"{{end}}>[[[Unlock]]]</button>
-  <button data-action="withdraw"{{if not $ready}} class="d-hide"{{end}}>[[[Withdraw]]]</button>
-  <button data-action="deposit"{{if not $ready}} class="d-hide"{{end}}>[[[Deposit]]]</button>
-  <button data-action="lock"{{if or (not $ready) (not $w.Encrypted)}} class="d-hide"{{end}}>[[[Lock]]]</button>
-  <button data-action="create" class="d-hide">[[[create_a_x_wallet]]]</button>
+    {{$ready := and $w.Running $w.Open}}
+    <button data-action="connect"{{if $w.Running}} class="d-hide"{{end}}>[[[Connect]]]</button>
+    <button data-action="unlock"{{if $w.Open}} class="d-hide"{{end}}>[[[Unlock]]]</button>
+    <button data-action="withdraw"{{if not $ready}} class="d-hide"{{end}}>[[[Withdraw]]]</button>
+    <button data-action="deposit"{{if not $ready}} class="d-hide"{{end}}>[[[Deposit]]]</button>
+    <button data-action="lock"{{if or (not $ready) (not $w.Encrypted)}} class="d-hide"{{end}}>[[[Lock]]]</button>
+    <button data-action="rescan"{{if not $w.Traits.IsRescanner}} class="d-hide"{{end}}>[[[Rescan]]]</button>
```

Comment:
The button never displays in this PR because no wallet's have this trait.  The button also has no bound action.  Both of those things are in https://github.com/decred/dcrdex/pull/1374.  this is just for illustration why we need a field in `WalletState` (or somewhere else in `SupportedAsset`).